### PR TITLE
docs(core/unittest): translate core/unittest

### PR DESCRIPTION
**Pull request by freebyron**
##### Checklist

- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change
翻译 core/unittest.md

### CODE REVIEW

**Comment by popomore** [comment]
这是准备翻译吗

**Comment by freebyron** [comment]
@popomore 是的

**Comment by popomore** [comment]
If you complete the translation, you can ping me to remove the label.

File: docs/source/en/core/unittest.md [hunk 1]
```diff
@@ -0,0 +1,712 @@
...
+
+```js
+// test/controller/home.test.js
+const assert = require('assert');
+const mock = require('egg-mock');
+
+describe('test/controller/home.test.js', () => {
+  let app;
+  before(() => {
+    // 创建当前应用的 app 实例
```
Comment (dead-horse) [inline docs/source/en/core/unittest.md]: translate to english

File: docs/source/en/core/unittest.md [hunk 2]
```diff
@@ -0,0 +1,712 @@
...
+  });
+
+  it('should US Dolar', () => {
+    const ctx = app.mockContext();
+    assert(ctx.helper.money(100) === '$ 100');
+  });
+});
+```l
+
+## Mock 方法
```
Comment (dead-horse) [inline docs/source/en/core/unittest.md]: need to be translated too

File: docs/source/en/core/unittest.md [hunk 3]
```diff
@@ -0,0 +1,712 @@
...
+Using `app.mockServiceError(service, methodName, error)` to mock exception.
+
+For example, mock the method `get(name)` in `app/service/user` to throw an exception.
+
+```js
+it('should mock service error', () => {
+  app.mockServiceError('user', 'get', 'mock user service error');
+  return app.httpRequest()
+    .get('/user?name=fengmk2')
+    // service 异常，触发 500 响应
```
Comment (dead-horse) [inline docs/source/en/core/unittest.md]: translate

**Comment by dead-horse** [comment]
others LGTM. fabulous job!

**Pull request closed without merging**